airgun/browser.py

```diff
@@ -477,6 +477,9 @@ class AirgunBrowserPlugin(DefaultPlugin):
         }
         '''
 
+    def ensure_page_safe(self, timeout='30s'):
+        super().ensure_page_safe(timeout)
```

Comment:
Pls use following format:
```python
return super(AirgunBrowserPlugin, self).ensure_page_safe(timeout)
```
Currently it's still technically possible to execute airgun & tests on py2, and this py2-uncompatible format doesn't bring much value to drop backwards compatibility.
Also please add small 1-linear docstring.

Reply:
@abalakh to be honest I don't think that we should support python 2 at all. One of the goals I set for AirGun was to be python 3 only but if we have folks using it with python 2, let me know so I can gently discourage them from doing so.

Reply:
@abalakh I disagree that we should write Python 2-compliant code just because a Jenkins job does not handle Python 3 yet. Let's get Kedar to update that process but let us not compromise on supporting Python 3 only, please?

Reply:
👍